components/post_markdown/index.test.ts

```diff
@@ -63,55 +68,55 @@ describe('makeGetMentionKeysForPost', () => {
                 myPreferences: {},
             },
         },
-    };
+    } as GlobalState;
 
     it('should return all mentionKeys', () => {
-        const post = {
+        const post = TestHelper.getPostMock({
             props: {
-                disable_group_highlight: false,
+                disable_group_highlight: true,
                 mentionHighlightDisabled: false,
             },
-        };
-        const getMentionKeysForPost = makeGetMentionKeysForPost(post, channel);
```

Comment:
why did we remove the parameters here

Reply:
![Uploading Screenshot 2022-10-01 at 8.24.54 PM.png…]()

Reply:
Oh sorry yes thats make function.